mewarpx/scripts_ancillary/tinyprofiler_plotting_example.py

```diff
@@ -0,0 +1,270 @@
+'''
+This script shows 3 examples of plotting the json data generated:
+- Plotting a stacked bar graph of the max/avg percents of functions for multiple runs.
+- Plotting the profiling data of a union of the top n expensive functions for multiple runs.
+- Plotting the profiling data of a specified collection of functions for multiple runs.
+
+As well as converting all the .out files in a directory to JSON files.
+'''
+
+import hatchet as ht
+import json
+import numpy as np
+import os
+import math
+from pathlib import Path
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import cm
```

Comment:
Please separate above imports into built-in imports - newline - third-party imports